App/NavigationRoot.js

```diff
@@ -19,7 +19,7 @@ const NavigationRoot = createSwitchNavigator(
     DistrictMatch: DistrictMatchStack
   },
   {
-    initialRouteName: 'Auth',
+    initialRouteName: 'App',
```

Comment:
Do you actually want to check this in, or is this just for convenience during development?

Reply:
did not mean to.  I'll fix it before I merge